frontend/src/components/common/chip/chip.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC } from 'react';
+import { ChipStyle } from 'common/enums/enums';
+import { getValidClasses } from 'helpers/helpers';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  value: string;
+  chipStyle?: ChipStyle;
```

Comment:
```suggestion
  chipStyle?: ChipStyle;
  chipColor??: ChipColor;
```
according to out design in figma